config/core/build-configs.yaml

```diff
@@ -368,7 +368,6 @@ fragments:
       mediatek/mt8192/scp.img
       mediatek/mt8195/scp.img
       mediatek/sof-tplg/sof-mt8186.tplg
-      mediatek/sof-tplg/sof-mt8195-mt6359-rt1019-rt5682-dts.tplg
```

Comment:
Do you expect we'll have to rely on `sof-mt8195-mt6359-rt1019-rt5682.tplg` once the driver support it? If that's the case, maybe simply replace the current filename with the non-dts one?

Reply:
That's what [2] suggests, however I haven't had time to test locally and validate that topology actually works on this machine.

Reply:
Right, so there are 2 options here, I'll let you choose the one you prefer:
* include `sof-mt8195-mt6359-rt1019-rt5682.tplg` right now as a substitute for future use
* merge this as-is, in which case I'll let you create and self-assign a new issue as a reminder to look into it once [2] gets merged

Reply:
I've updated to option 1 so I don't need to remember to update the firmware later.